Expand Difficulty Increase mod articles (#12629)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Blinds/en.md

*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Blinds** mod covers the entire screen with [shoji doors](https://en.wikipedia.org/wiki/Shoji) without the possibility to see the objects behind. The more successive misses you get, the more the doors opens. In contrast, building a combo will have the opposite effect.

Reviewer: ```suggestion
The **Blinds** mod covers the entire screen with [shoji doors](https://en.wikipedia.org/wiki/Shoji) without the possibility to see the objects behind. The more successive misses the player gets, the more the doors open. Building up a combo will have the opposite effect.
```
